### PR TITLE
Allows the IMS to be used for necrosis surgery (again)

### PR DESCRIPTION
**Pull request by Hughgent**
🆑 Hughgent
tweak: Allows the IMS to be used for necrosis.
/🆑

This got lost in the rebase, so here it is again.

### CODE REVIEW

**Comment by Rohesie** [comment]
Have you properly tested this?

**Comment by spookydonut** [comment]
This is fine it's just a revert of this from 10 months ago: https://github.com/tgstation/TerraGov-Marine-Corps/commit/d3a8a0dde635ae1ba5c79a8bc57d6afd4f62dc54#diff-bc6648424256142b3afcd3cba72dd36a

File: code/modules/surgery/necrosis.dm [hunk 1]
```diff
@@ -14,7 +14,7 @@
 /datum/surgery_step/necro/fix_dead_tissue //Debridement
 
 	allowed_tools = list(
-		/obj/item/tool/surgery/scalpel/manager = 0,			\
+		/obj/item/tool/surgery/scalpel/manager = 100,			\
```
Comment (spookydonut) [inline code/modules/surgery/necrosis.dm]: Actually no, just remove this line.

**Comment by Hughgent** [comment]
oh and if anyone says "I can't do the Necrosis surgery!" tell them to finish up inside the rib cage and then to close the rib cage (retractor - bone gel) before attempting again. (and also sterilize yourself you filthy animal.)

**Pull request merged**